content/developer/composer/_index.md

```diff
@@ -53,10 +53,10 @@ You can import a `.yaml` file that contains definitions of multiple node types.
 
```

Comment:
Change "Cloudify Manager Cloudify Console" in _If you are already using Cloudify Manager Cloudify Console, you can click the **Cloudify Composer** button in the **Local Blueprints** page._ to "Cloudify Console".